core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/compare/CompareSetResult.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2020 B2i Healthcare. All rights reserved.
+ *******************************************************************************/
+package com.b2international.snowowl.core.compare;
+
+import java.util.Set;
+
+import com.google.common.collect.ListMultimap;
+/**
+ * @since 7.8
+ */
+public class CompareSetResult <T> {
```

Comment:
Make this class final.
Rename it to `ConceptMapCompareResult`.
What can be `T` here? 
This is too generic, and it won't work well in REST APIs with Swagger.
Either we create abstract superclass with `T` type parameter and concrete implementations for each possible `T` type (like `CollectionResource`) or introduce a generic interface for type `T`.

Reply:
Done